docs/application/native/guides/internationalization/i18n.md

```diff
@@ -114,8 +118,8 @@ A Ucalendar object can produce all the time field values needed to implement the
 
 When computing a Udate from the time fields, 2 special circumstances can arise. The information can be insufficient to compute the Udate (you have only the year and the month, but not the day of the month), or the information can be inconsistent (such as "Tuesday, July 15, 1996" even though July 15, 1996 is actually a Monday).
 
-- **Insufficient information**The calendar uses the default information to specify the missing fields. This can vary by calendar: for the Gregorian calendar, the default for a field is the same as that of the start of the epoch, such as `I18N_UCALENDAR_YEAR = 1970`, `I18N_UCALENDAR_MONTH = JANUARY`, `I18N_UCALENDAR_DATE = 1`.
-- **Inconsistent information**If the fields conflict, the calendar prefers the most recently set fields. For example, when determining the day, the calendar looks for one of the following field combinations listed in the following table. The most recent combination, as determined by the most recently set single field, is used.
+- **Insufficient information**: The calendar uses the default information to specify the missing fields. This can vary by calendar. For example, the Gregorian calendar, the default for a field is the same as that of the start of the epoch, such as `I18N_UCALENDAR_YEAR = 1970`, `I18N_UCALENDAR_MONTH = JANUARY`, `I18N_UCALENDAR_DATE = 1`.
+- **Inconsistent information**: If there is field’s conflict, the calendar prefers the most recently set fields. For example, when determining the day, the calendar looks for one of the following field combinations listed in the following table. The most recent combination, as determined by the most recently set single field, is used.
```

Comment:
Line 222: Please change check to verify and add colon at the end of the sentence:

To verify whether a character is in a specific character class, use the `i18n_uchar_is_xxx()` function corresponding to the binary property in the following table